resource-service/README.md

```diff
@@ -15,31 +19,45 @@ to any Git-based service such as GitLab, GitHub, Bitbucket, etc.
 ------------          ------------          ------------
   1 \                   1  \                   1  \
      \ *                    \ *                    \ *
-   ------------           ------------           ------------ 
-   |          |           |          |           |          | 
-   | Resource |           | Resource |           | Resource |  
-   |          |           |          |           |          |  
-   ------------           ------------           ------------ 
+   ------------           ------------           ------------
+   |          |           |          |           |          |
+   | Resource |           | Resource |           | Resource |
+   |          |           |          |           |          |
+   ------------           ------------           ------------
 ```
 
 ## Installation
 
-The *resource-service* is installed as a part of [keptn](https://keptn.sh)
+The *resource-service* replaces the *configuration-service*, hence only one of the two can be run at the same time.
+The *resource-service* can be enabled during the installation of Keptn setting the Helm value `control-plane.resourceService.enabled` to `true`.
+This flag changes the *configuration-service* `Service` to point towards the *resource-service* `Pod`.
+When the resource-service is enabled by default, the Kubernetes `Service` will change name to `resource-service`.
 
-## Deploy in your Kubernetes cluster
+### Deploy it directly into your Kubernetes cluster
 
-To deploy the current version of the *resource-service* in your Keptn Kubernetes cluster, 
+To deploy the current version of the *resource-service* in your Keptn Kubernetes cluster,
 use the file `deploy/service.yaml` from this repository and apply it.
 
 ```console
 kubectl apply -f deploy/service.yaml
 ```
 
-## Delete in your Kubernetes cluster
+### Delete it from your Kubernetes cluster
 
-To delete a deployed *configuration-service*, use the file `deploy/service.yaml` from this repository 
+To delete a deployed *resource-service*, use the file `deploy/service.yaml` from this repository
 and delete the Kubernetes resources:
 
 ```console
 kubectl delete -f deploy/service.yaml
 ```
+
+## Migration from the configuration-service
+
+Before migrating from the *configuration-service* to the *resource-service* it is recommended to (i) attach an upstream to your Keptn projects and (ii) do a [backup](https://keptn.sh/docs/0.15.x/operate/backup_and_restore/#back-up-configuration-service). If you set an upstream for all your Keptn projects, no additional steps are required.

```

Comment:
... it is recommended to ...

shouldn`t it be 

.. it is required to ... ?

Reply:
Both setting the upstream and doing the backup are not strictly mandatory